views/trust-individual-beneficial-owner.html

```diff
@@ -0,0 +1,102 @@
+{% extends "layout.html" %}
+
+{% set title = "Tell us about an individual (Trusts)" %}
+
+{% block pageTitle %}
+  {% include "includes/page-title.html" %}
+{% endblock %}
+
+{% block backLink %}
+  {% include "includes/back-link.html" %}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {% include "includes/list/errors.html" %}
+
+      <span class="govuk-caption-l">{{ trustData.trustName }}</span>
+      <h1 class="govuk-heading-xl govuk-!-margin-0">{{ pageParams.title }}</h1>
+      <br> <p class="govuk-body">You can add more later.</p> <br>
+
+      <form class="form" method="post">
+
+        {% set interestedPersonDateHtml %}
+        <br>
+        {% set fieldParam = {
+          name: 'interestedPersonDate',
+          label: 'What date did they become an interested person?',
+          classes: 'govuk-fieldset__legend--s',
+          error: errors.interestedPersonDate
+        } %}
+        {% include "includes/inputs/date-input.html" %}
```

Comment:
not critical now, but for date fields you should pass values
```
 value: {
            'Day': formData.createdDateDay,
            'Month': formData.createdDateMonth,
            'Year': formData.createdDateYear
          }
```

Reply:
I think maybe keep this as is because Okoye is currently working on mapping story based off this branch so would have had to add them anyway